Source/Scene/GltfLoader.js

```diff
@@ -883,6 +858,33 @@ function loadAttribute(
       attribute.byteOffset = 0;
       attribute.byteStride = undefined;
       attribute.quantization = vertexBufferLoader.quantization;
+
+      if (loadBuffer) {
+        attribute.buffer = vertexBufferLoader.buffer;
+      }
+
+      if (loadTypedArray) {
+        attribute.typedArray = ComponentDatatype.createArrayBufferView(
+          vertexBufferLoader.quantization.componentDatatype,
+          vertexBufferLoader.typedArray.buffer
+        );
+      }
```

Comment:
This refactor exposed a bug related to packed typed arrays of Draco content. Basically `getPackedTypedArray` is not quantization aware and it fails with an out of bounds error.

Here Draco uses its own method to create the typed array.